src/components/Navbar.css

```diff
@@ -0,0 +1,38 @@
+header {
+  background-color: white;
+  display: flex;
+  justify-content: space-between;
+  padding-top: 15px;
+  padding-left: 5%;
+  padding-right: 5%;
+  padding-bottom: 15px;
+  box-shadow: 0 2px 5px -5px #888;
+}
+
+.header-left {
+  display: flex;
+}
+
+.header-title {
+  margin: 0 !important;
+  color: #0290ff;
+  height: min-content;
+  font-size: 1rem;
```

Comment:
- [ ] It would be nicer to use the same font sizes as the one provided on [Zeplin](https://zeplin.io/) to improve your UI design.

Reply:
Thank you for your feed back but I have a question , I have already send you it on slake .